lists-ui/src/views/List/components/Actions.tsx

```diff
@@ -188,14 +190,15 @@ export function Actions({
     modals.open({
       title: (
         <Text fw='bold' size='lg'>
-          Editing {meta.title}
+          {intl.formatMessage({id:'actions.editMetadata.title', defaultMessage:'Edit List Metadata'})}
```

Comment:
Deliberately removed {meta.title} as its not needed in a modal popup and clutters UI.